test/utils.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  expect
+} from 'bun:test';
+/**
+ * exported helper functions to make up for bun's wiptest
+ * lacking some features
+ */
+
+
+export const isClose = (actual: number, expected: number, error = 0.001) => {
```

Comment:
That last change, Typescript will actually automatically remove the type  as it can infer the type of `number` from the default of 0.001. So I'm not sure this change is necessary, but if you'd prefer to have the type explicitly defined, I can definitely do so!

Reply:
Yeah, I haven't touched ts in a few years; I guess having the type makes it future proof unless we remove the default, but either way